### DISCUSSION TITLE
BERLIN Quantum BBQ 6th Edition [Berlin, DE] (2024-09-10)

### DISCUSSION THREAD

**Discussion by Itaborala** [Quantumland-art/community#70] [Events]
The company [OpTecBB](https://optecbb.de/) organises a monthly BBQ gathering of industries and startups in quantum based in Berlin.
At occasions these gatherings also contain talks about various topics of quantum technologies (not only quantum computing).

At the 6th edition - September 10, 18:30 - The topic of discussion is Quantum Computer Music. I'm pasting the website announcement below, together with the link for the event and for registering (for free):

> [Event announcement (in german)](https://optecbb.de/veranstaltungen/veranstaltung/bar-bq-2082/)

> [Tickets (free entrance, registration needed)](https://www.eventbrite.de/e/berlin-quantum-bbq-6-quantum-music-tickets-1004342343987)

### _**Location: Rudower Chaussee 29, 6. Floor - 12489 Berlin  (Near S-Bahn Adlershof)**_

-----------------------------------------------
### BERLIN QUANTUM BBQ #6 organized by our friends from OpTecBB!

It's time for the 6th edition of the **BERLIN Quantum BBQ** at **Leap Coworking**! Get ready to mingle with fellow quantum enthusiasts, enjoy delicious BBQ, and listen to exciting talks about the latest developments in quantum technology.

For this edition we want to welcome the creativity and cultural side of science with Quantum [_Computer_] Music. Paulo Itaborai (Karl Jansen, Quantum Computing, DESY) will present a perspective on music creation based on quantum using different concepts (Qubits, Live coding, algorithms), including a music performance.

This event series is all about building community and sharing knowledge in a laid-back atmosphere. Whether you are a quantum expert or just curious about this cutting-edge field, you are welcome to join us for an evening of fun, networking, and learning.

Mark your calendars and don't miss out on this unique opportunity to connect with the quantum community in Berlin!

**Comment by omarcostahamido** [comment]
Haha, nice! Do you know what you will be performing? Or is it demo for the
talk?

**Comment by Itaborala** [comment]
It will be a 1-person demo version of **_Hexagonal Chambers_** by Cephas Teom & Itabora Puy (2023-24)
